aimmo-reverse-proxy/nginx.conf

```diff
@@ -0,0 +1,13 @@
+
+http {
+    resolver 10.0.0.10;
+    server {
+        listen 80;
+        location ~^/game/([a-zA-Z]+)/? {
+            proxy_pass "http://game-$1.default.svc.cluster.local/";
```

Comment:
It's generally better not to hardcode the cluster dns name, or the namespace - the search path should be set up correctly to allow for this.

e.g.
```
proxy_pass "http://game-$1/";
```